src/app/components/user-details/user-details.component.html

```diff
@@ -0,0 +1,89 @@
+<div class="user-detials-wrapper">
+    <mat-card class="user-card">
+        <mat-card-header>
+            <mat-icon mat-card-avatar>account_circle</mat-icon>
+            <mat-card-title>{{user.name}}</mat-card-title>
+            <mat-card-subtitle>{{user.position}}</mat-card-subtitle>
+
+            <span class="spacer"></span>
+
+            <mat-card-actions>
+                <button mat-flat-button color="primary" (click)="toggleIsView()">{{!isView ? "View Details" : "Edit Profile"}}</button>
+            </mat-card-actions>
+        </mat-card-header>
+        <mat-card-content>
+
+            <div *ngIf="isView; then thenTemplateName else elseTemplateName"></div>
```

Comment:
This would look more readable written the following way: 
<div *ngIf="isView else editModeTemplate>
//add the template for #thenTemplateName (but without the <ng-template> tag)
</div>
elseTemplateName should be renamed to editModeTemplate. Use self describing variable names.